crates/next-core/src/next_shared/webpack_rules/mod.rs

```diff
@@ -71,6 +72,28 @@ impl WebpackLoaderBuiltinCondition {
             Self::EdgeLight => "edge-light",
```

Comment:
I'm just matching what we had before here, but I think the name "edge light" is potentially confusing to end-users, so we could expose this as "edge" instead.

Now would be the time to rename it if we want to.